docs/source/basic/install.rst

```diff
@@ -82,8 +82,8 @@ This allows the usage of the coresponding executor e.g. `gpuCuda`, `gpuHip` or `
   spack load cmake@3.29.1
 
   # -Dalpaka_DEP_OMP=ON is implicitly set, if the compiler not support OpenMP only serial code will be generated
-  cmake ../alpaka -Dalpaka_TESTING=ON -Dalpaka_BENCHMARKS=ON -Dalpaka_EXAMPLES=ON -DBUILD_TESTING=ON
-  cmake --build ../alpaka3./alpaka3 -j
+  cmake ../alpaka3 -Dalpaka_TESTING=ON -Dalpaka_BENCHMARKS=ON -Dalpaka_EXAMPLES=ON -DBUILD_TESTING=ON
+  cmake --build ../alpaka3 -j
```

Comment:
this should be `cmake --build . --parallel` it was before `make -j` but which the switch to read the docs I changed it to clean CMake but not sure why I wrote bullshit.